# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../boost/notools.patch.cross
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- boost_1_40_0/tools/CMakeLists.txt.orig       2009-09-06 10:39:01.000000000 +0200
+++ boost_1_40_0/tools/CMakeLists.txt    2009-09-06 10:51:13.000000000 +0200
@@ -85,5 +85,5 @@

 # add_subdirectory(quickbook)
 # add_subdirectory(wave)
-add_subdirectory(bcp)
-add_subdirectory(inspect)
\ No newline at end of file
+#add_subdirectory(bcp)
+#add_subdirectory(inspect)
